### PR TITLE
fixes #803

### LINKED ISSUE #803: SSO Booster / Spring Boot: application.properties out of date
Current application.properties has more interpolation. Also callout 2 mentions PEM format without explanation: either expand the acronym and explain or remove the format mention since it doesn't really add any interesting information without more details.

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

**Comment by rhoads-zach** [comment]
@metacosm mind taking a look?

File: docs/topics/secured-mission-springboot-adapter.adoc [hunk 1]
```diff
@@ -9,26 +9,25 @@ Both the SSO adapter and endpoint security are configured in `src/main/resources
...
-keycloak.credentials.secret=1daa57a2-b60e-468b-a3ac-25bd2dc2eadc <5>
-keycloak.use-resource-role-mappings=true <6>
-keycloak.bearer-only=true <7>
+keycloak.realm=${realm:master} <1>
+keycloak.realm-key=...
+keycloak.auth-server-url=${sso.auth.server.url} <2>
+keycloak.resource=${client.id:secured-booster-endpoint} <3>
+keycloak.credentials.secret=${secret:1daa57a2-b60e-468b-a3ac-25bd2dc2eadc} <4>
+keycloak.use-resource-role-mappings=true <5>
+keycloak.bearer-only=true <5>
```
Comment (tradej) [inline docs/topics/secured-mission-springboot-adapter.adoc]: I believe this should be `<6>`.

Comment (rhoads-zach) [reply docs/topics/secured-mission-springboot-adapter.adoc]: nice catch @tradej 
fixed

Comment (tradej) [reply docs/topics/secured-mission-springboot-adapter.adoc]: Thanks, I only discovered it because the validation failed. :)

**Comment by tradej** [comment]
Merging...

**Comment by tradej** [comment]
merged 83fec9bc2

**Pull request merged**